themes/base/theme.css

```diff
@@ -134,7 +134,7 @@ a.ui-button:active,
 .ui-widget-content .ui-state-highlight,
 .ui-widget-header .ui-state-highlight {
 	border: 1px solid #dad55e/*{borderColorHighlight}*/;
-	background: #fffa90/*{bgColorHighlight}*/ /*{bgImgUrlHighlight}*/ /*{bgHighlightXPos}*/ /*{bgHighlightYPos}*/ /*{bgHighlightRepeat}*/;
+	background-color: #fffa90/*{bgColorHighlight}*/ /*{bgImgUrlHighlight}*/ /*{bgHighlightXPos}*/ /*{bgHighlightYPos}*/ /*{bgHighlightRepeat}*/;
```

Comment:
> @scottgonzalez how can I test that?

Testing it is a bit tricky. It would require setting up https://github.com/jquery/download.jqueryui.com/ locally and then using your updated theme in ThemeRoller.

> Is there a way to make ThemeRoller produce individual properties, rather than consolidated?

You'd have to list all the properties. ThemeRoller just does token replacement. But won't producing individual properties cause the same problem?